packages/pn-personafisica-webapp/src/components/Contacts/DigitalContactElem.tsx

```diff
@@ -159,6 +170,7 @@ const DigitalContactElem = forwardRef<{ editContact: () => void }, Props>(
             sx={{
               wordBreak: 'break-word',
             }}
+            id={f.id}
```

Comment:
the ids in this file are not unique because this component is used multiple times in contact page

Reply:
same as before

Reply:
same as before